viite-UI/src/view/ProjectEditForm.js

```diff
@@ -483,6 +483,10 @@
         }
       });
 
+      rootElement.on('change', '#trackCodeDropdown', function () {
```

Comment:
Quick question @plourenco90 why is checkInputs being called with a ('.project-') parameter when the function accepts no input?

Reply:
Its because this method is used in the split validation as well, so this is the way it differs from witch form is validating